.travis.yml

```diff
@@ -40,6 +40,19 @@ install:
     - cd test-server && pip install -r requirements.txt && cd ..
     - ./node_modules/.bin/webdriver-manager update
 
+before_script: >
+    df -h
```

Comment:
I'll explain, but I hope we'll switch from Travis and we'll remove this file after "Unify deployment" task